examples/pytorch/image_recognition/se_resnext/quantization/ptq/fx/README.md

```diff
@@ -0,0 +1,53 @@
+Step-by-Step
+============
+
+This document describes the step-by-step instructions for reproducing PyTorch se_resnext tuning and benchmarking results with Intel® Neural Compressor.
+
+> **Note**
+>
+> * PyTorch quantization implementation in imperative path has limitation on automatically execution. It requires manually adding QuantStub and DequantStub for quantizable ops, and  also fusion operation manually.
+> * Intel® Neural Compressor supposes user have done these two steps before invoking Intel® Neural Compressor interface.
+>   For details, please refer to https://pytorch.org/docs/stable/quantization.html
+
+# Prerequisite
+## 1. Environment
+Python 3.6 or higher version is recommended.
+The dependent packages are all in requirements, please install as following.
+```shell
+cd examples/pytorch/image_recognition/se_resnext/quantization/ptq/fx
+pip install -r requirements.txt
+```
+## 2. Install Model
+```shell
+python setup.py install
```

Comment:
Can we also use git clone rather than ship all the code? So we only need the main scripts and readme.

Reply:
done